README.md

```diff
@@ -4,30 +4,35 @@ This OS is based on the excellent blog of [phil-opp](https://os.phil-opp.com/).
 
 ### Structure
 
-TODO
+The project is divided into multiple folders:
```

Comment:
Can you add relative links to the folders like [rost-lib](rost-lib/)

Reply:
Done